excerise.go

```diff
@@ -0,0 +1,61 @@
+package main
+import "fmt"
+func worker(id int, jobs <-chan int, results chan<- int) {
+						
+for j := range jobs { go func() {
```

Comment:
Anonymous goroutine manipulates with variable `j` but it is not defined inside and its not passed from outside.
Perhaps a quick fix can be to pass `j` to the anon goroutine.
```
		go func(j int) {
			switch j % 3 { 
				case 0:
					j=j*1 
				case 1:
					j=j*2
					results <- j * 2 
				case 2:
					results <- j * 3
					j=j*3 
			}
		}(j)
```